protocols/virtualfund/virtualfund_single_hop_test.go

```diff
@@ -4,15 +4,14 @@ import (
 	"bytes"
 	"fmt"
 	"math/big"
-	"path/filepath"
 	"reflect"
-	"runtime"
 	"testing"
 
 	"github.com/statechannels/go-nitro/channel"
 	"github.com/statechannels/go-nitro/channel/consensus_channel"
 	"github.com/statechannels/go-nitro/channel/state"
 	"github.com/statechannels/go-nitro/channel/state/outcome"
+	helpers "github.com/statechannels/go-nitro/internal/testhelpers"
```

Comment:
Is this a reasonable practice here?

```suggestion
	. "github.com/statechannels/go-nitro/internal/testhelpers"
```

Reply:
That makes sense to me! I've applied that in https://github.com/statechannels/go-nitro/pull/529/commits/eceefe6ee3fd6c794c263028437af8bf7492d0bf